client/src/components/Chat.jsx

```diff
@@ -45,6 +45,7 @@ let senderId;
 
 const Chat = () => {
 	const { app } = useApp();
+	const { settings } = app;
```

Comment:
No need to use the settings set the correct color theme, we use tailwindCSS and already use the Tailwind way of setting light or dark themes.

Here's the docs: https://tailwindcss.com/docs/dark-mode

This is where we activate the dark mode: https://github.com/Dun-sin/Whisper/blob/58396be5d2db76fac6b7f84a8cb7bc4de9b602f0/client/src/App.jsx#L98